samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -94,3 +94,16 @@ tests:
       - mg100
     integration_platforms:
       - nrf52840dk_nrf52840
+  sample.mcumgr.smp_svr.bt.nrf54l15pdk.ext_flash:
+    extra_args:
+      - OVERLAY_CONFIG="overlay-bt.conf"
+      - DTC_OVERLAY_FILE="boards/nrf54l15pdk_nrf54l15_cpuapp_ext_flash.overlay"
+      - mcuboot_CONF_FILE="boards/nrf54l15pdk_nrf54l15_cpuapp_ext_flash.conf"
+      - mcuboot_EXTRA_DTC_OVERLAY_FILE="boards/nrf54l15pdk_nrf54l15_cpuapp_ext_flash.overlay"
+    extra_configs:
+      - CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=y
+      - CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK=y
```

Comment:
`CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK` can be probable moved to boards/nrf54l15pdk_nrf54l15_cpuapp_ext_flash.conf, it is enough to set it in mcuboot kconfigs

Reply:
Behaviour you observed is not documented. It is described that this should be set for the application.